lib/matplotlib/backends/backend_webagg_core.py

```diff
@@ -243,7 +243,7 @@ def get_renderer(self, cleared=None):
         # Mirrors super.get_renderer, but caches the old one
         # so that we can do things such as produce a diff image
         # in get_diff_image
-        _, _, w, h = self.figure.bbox.bounds
+        _, _, w, h = np.round(self.figure.bbox.bounds).astype(int)
         w, h = int(w), int(h)
```

Comment:
This conversion to python int is not done in the other two backends, so presumably it is not needed here.  (I see that numpy.int64 and int instances both hash to the corresponding python int.)